python/src/scipp/plot/plot_1d.py

```diff
@@ -99,6 +98,35 @@ def __init__(self,
                          config.plot.height / config.plot.dpi),
                 dpi=config.plot.dpi)
 
+        # Determine whether error bars should be plotted or not
+        self.variances = {}
+        for name, var in self.scipp_obj_dict.items():
+            self.variances[name] = var.variances is not None
+        if variances is not None:
+            if isinstance(variances, bool):
+                for name, var in self.scipp_obj_dict.items():
+                    self.variances[
+                        name] = variances and var.variances is not None
```

Comment:
Can just iterate the dict and do `&= variances` on each value?

Reply:
I don't think I can just do that because I also need to check that `var.variances` exists (i.e. is not `None`)

Reply:
Didn't you do that already 5 lines above?

Reply:
:man_facepalming: